geomesa-filter/src/test/scala/org/locationtech/geomesa/filter/function/FastPropertyTest.scala

```diff
@@ -49,7 +49,7 @@ class FastPropertyTest extends Specification {
       val sf = new ScalaSimpleFeature("id", sft)
       sf.setAttributes(Array[AnyRef]("myname", "POINT(45 45)"))
 
-      val filter = ECQL.toFilter("testns:name = 'myname'", FastFilterFactory.factory)
+      val filter = FastFilterFactory.toFilter(sft, "testns:name = 'myname'")
```

Comment:
i think it should still work - FastPropertyName was only instantiated through FastFilterFactory, so it would get invoked in the same way here, just the logic is calculated before creating the filter instead of after. `ECQL.toFilter` wasn't ever hitting that logic.